src-11ty/faq.html

```diff
@@ -85,7 +91,7 @@ <h1 class="card-header">Frequently Asked Questions</h1>
   </p>
 </div>
 
-<div class="section-header" id="supported-pages">What pages are supported?</div>
```

Comment:
Why no longer the id?

Reply:
We no longer link to it from the index.

Reply:
We do from the frontend

Reply:
![image](https://user-images.githubusercontent.com/5662298/109545802-9a428d00-7ac9-11eb-9f87-bef6989cece9.png)

Reply:
Oh let's reinstate.